components/Indifferent.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Components;
+
+/**
+ * Interface for objects on which any method can be called.
+ */
+interface Indifferent
+{
+
+    /**
+     * Handles any method call.
+     *
+     * @param string $method
+     * @param array<mixed> $args
+     * @return mixed
+     */
+    public function __call(string $method, array $args): mixed;
+}
```

Comment:
This interface is not needed yet but will probably be used in the future. Although it is dead code, it is an interface, so this is not a serious issue.